src/cmd-generate-release-meta

```diff
@@ -170,7 +182,11 @@ def append_build(out, input_):
     # KubeVirt specific additions: https://github.com/coreos/stream-metadata-go/pull/41
     if input_.get("kubevirt", None) is not None:
         arch_dict["media"].setdefault("kubevirt", {}).setdefault("image", {})
-        arch_dict["media"]["kubevirt"]["image"].update(input_.get("kubevirt", {}))
+        tag = get_floating_tag(input_["buildid"], input_["kubevirt"]["tags"])
```

Comment:
Optional: I feel like this could use a comment? Or I guess we can just rely on git history.